src/main/java/appeng/core/features/AEFeature.java

```diff
@@ -81,15 +87,28 @@ public boolean isVisible()
 	ExportBus( Constants.CATEGORY_NETWORK_BUSES ),
 	StorageBus( Constants.CATEGORY_NETWORK_BUSES ),
 	PartConversionMonitor( Constants.CATEGORY_NETWORK_BUSES ),
+	ToggleBus( Constants.CATEGORY_NETWORK_BUSES ),
+	Panels( Constants.CATEGORY_NETWORK_BUSES ),
+	QuartzFiber( Constants.CATEGORY_NETWORK_BUSES ),
+	CableAnchor( Constants.CATEGORY_NETWORK_BUSES ),
 
 	PortableCell( Constants.CATEGORY_PORTABLE_CELL ),
 
 	StorageCells( Constants.CATEGORY_STORAGE ),
 	MEChest( Constants.CATEGORY_STORAGE ),
 	MEDrive( Constants.CATEGORY_STORAGE ),
 	IOPort( Constants.CATEGORY_STORAGE ),
+	Condenser( Constants.CATEGORY_STORAGE ),
 
 	NetworkTool( Constants.CATEGORY_NETWORK_TOOL ),
+	MemoryCard( Constants.CATEGORY_NETWORK_TOOL ),
+
+	GlassCables( Constants.CATEGORY_CABLES ),
+	CoveredCables( Constants.CATEGORY_CABLES ),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)